connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoaderTest.java

```diff
@@ -22,6 +22,7 @@
 import org.junit.rules.TemporaryFolder;
 
 import java.io.File;
+import java.io.IOException;
```

Comment:
There's one more leaked classloader, in testLoadingMixOfValidAndInvalidPlugins if you want to fix that as well.
I don't see these warnings as relevant, since this is test code. But I won't stop you from adding the try-with-resources.

Reply:
yeah, won't hurt :)